qa/vagrant/src/main/java/org/elasticsearch/packaging/test/PackagingTestCase.java

```diff
@@ -21,31 +21,54 @@
 
 import com.carrotsearch.randomizedtesting.JUnit3MethodProvider;
 import com.carrotsearch.randomizedtesting.RandomizedRunner;
+import com.carrotsearch.randomizedtesting.annotations.TestCaseOrdering;
 import com.carrotsearch.randomizedtesting.annotations.TestMethodProviders;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.elasticsearch.packaging.util.Distribution;
+import org.elasticsearch.packaging.util.Installation;
+import org.junit.Assert;
 import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.Rule;
 import org.junit.rules.TestName;
 import org.junit.runner.RunWith;
 
+import static org.elasticsearch.packaging.util.Cleanup.cleanEverything;
+import static org.junit.Assume.assumeTrue;
+
 @RunWith(RandomizedRunner.class)
 @TestMethodProviders({
     JUnit3MethodProvider.class
 })
+@TestCaseOrdering(TestCaseOrdering.AlphabeticOrder.class)
 /**
  * Class that all packaging test cases should inherit from. This makes working with the packaging tests more similar to what we're
  * familiar with from {@link org.elasticsearch.test.ESTestCase} without having to apply its behavior that's not relevant here
  */
-public abstract class PackagingTestCase {
+public abstract class PackagingTestCase extends Assert {
```

Comment:
What's the motivation for extending `Assert`?

Reply:
It makes assert methods available without needing to do imports. This is what LuceneTestCase does (and by extension, ESTestCase).